src/apis/tokenlistapi.py

```diff
@@ -0,0 +1,50 @@
+"""
+TokenListAPI for fetching a curated token list.
+"""
+# pylint: disable=logging-fstring-interpolation
+from typing import Optional
+import requests
+from src.helper_functions import get_logger
+from src.constants import (
+    header,
+    REQUEST_TIMEOUT,
+)
+
+
+class TokenListAPI:
+    """
+    Class for fetching a curated token list.
+    """
+
+    def __init__(self) -> None:
+        self.logger = get_logger()
+
+    def get_token_list(self) -> Optional[list[str]]:
+        """
+        Returns a token list.
+        """
+        kleros_url = "http://t2crtokens.eth.link"
+        one_inch_url = "https://tokens.1inch.eth.link"
+        aave_url = "https://tokenlist.aave.eth.link"
+
+        url_list = [kleros_url, one_inch_url, aave_url]
+
+        token_list: list[str] = []
+        for url in url_list:
```

Comment:
```suggestion
        self.logger = get_logger()
        self.token_lists = [
            "http://t2crtokens.eth.link",
            "https://tokens.1inch.eth.link",
            "https://tokenlist.aave.eth.link",
        ]

    def get_token_list(self) -> Optional[list[str]]:
        """
        Returns a token list.
        """
        token_list: list[str] = []
        for url in self.token_lists:
```

Looks a bit cleaner to pull the token list into the class. But both work.